lib/filesystem.nix

```
@@ -54,4 +54,44 @@
       dir + "/${name}"
   ) (builtins.readDir dir));
 
+  # findSuffix: String -> Path -> [ Path ]
```

```suggestion
  # findBySuffix: String -> Path -> [ Path ]
```

This mirrors `lib.sources.sourceByRegex`, `sourceFilesBySuffices`